packages/odata-common/src/request-builder/batch/batch-request-builder.ts

```diff
@@ -37,6 +39,9 @@ export class BatchRequestBuilder<
     )[] // readonly entityToConstructorMap: Record< //   string, //   EntityApi<EntityBase, DeSerializersT> // >
   ) {
     super(new ODataBatchRequestConfig(defaultServicePath));
+    this.deSerializers = Object.values(
+      this.getEntityToApiMap()
+    )[0].deSerializers;
```

Comment:
Could this fail? Maybe it's safer to use `.find()` instead of accessing by index.

Reply:
in principle if there are no request the whole thing does nothing and the undefined will not hurt but lets use find because it is always nicer than array access